ipapython/config.py

```diff
@@ -129,131 +118,8 @@ def verify_args(parser, args, needed_args = None):
-    servers = []
-    try:
-        if not config.default_domain:
-            # try once with REALM -> domain
-            domain = str(config.default_realm).lower()
-            name = "_ldap._tcp." + domain
-
-            try:
-                servers = resolver.query(name, rdatatype.SRV)
-            except DNSException:
-                # try cycling on domain components of FQDN
-                try:
-                    domain = dns.name.from_text(socket.getfqdn())
-                except DNSException:
-                    return False
-
-                while True:
-                    domain = domain.parent()
-
-                    if str(domain) == '.':
-                        return False
-                    name = "_ldap._tcp.%s" % domain
-                    try:
-                        servers = resolver.query(name, rdatatype.SRV)
-                        break
-                    except DNSException:
-                        pass
-
-            config.default_domain = str(domain).rstrip(".")
-
-        if discover_server:
-            if not servers:
-                name = "_ldap._tcp.%s." % config.default_domain
-                try:
-                    servers = resolver.query(name, rdatatype.SRV)
-                except DNSException:
-                    pass
-
-            for server in servers:
-                hostname = str(server.target).rstrip(".")
-                config.default_server.append(hostname)
-
-    except Exception:
-        pass
-
 def add_standard_options(parser):
     parser.add_option("--realm", dest="realm", help="Override default IPA realm")
     parser.add_option("--server", dest="server",
                       help="Override default FQDN of IPA server")
     parser.add_option("--domain", dest="domain", help="Override default IPA DNS domain")
-
-def init_config(options=None):
-    if options:
-        config.default_realm = options.realm
-        config.default_domain = options.domain
-        if options.server:
-            config.default_server.extend(options.server.split(","))
-
-    if len(config.default_server):
-        discover_server = False
-    else:
-        discover_server = True
-    __parse_config(discover_server)
-    __discover_config(discover_server)
-
-    # make sure the server list only contains unique items
-    new_server = []
-    for server in config.default_server:
-        if server not in new_server:
-            new_server.append(server)
-    config.default_server = new_server
-
-    if not config.default_realm:
-        raise IPAConfigError("IPA realm not found in DNS, in the config file (/etc/ipa/default.conf) or on the command line.")
-    if not config.default_server:
-        raise IPAConfigError("IPA server not found in DNS, in the config file (/etc/ipa/default.conf) or on the command line.")
-    if not config.default_domain:
-        raise IPAConfigError("IPA domain not found in the config file (/etc/ipa/default.conf) or on the command line.")
```

Comment:
With this deletion, there's nothing that raises IPAConfigError. Is there a reason to leaving it there?